src/Uno.UI/UI/Xaml/Controls/Popup/Popup.wasm.cs

```diff
@@ -61,23 +61,36 @@ protected override void OnIsOpenChanged(bool oldIsOpen, bool newIsOpen)
 				}
 			}
 
+			bool pressed = false;
+
 			if (previousPanel != null)
 			{
-				previousPanel.PointerPressed -= Panel_PointerPressed;
+				previousPanel.PointerPressed -= pointerPressed;
+				previousPanel.PointerReleased -= pointerReleased;
 			}
 			if (newPanel != null)
 			{
-				newPanel.PointerPressed += Panel_PointerPressed;
+				newPanel.PointerPressed += pointerPressed;
+				newPanel.PointerReleased += pointerReleased;
 			}
-		}
 
-		private void Panel_PointerPressed(object sender, Input.PointerRoutedEventArgs e)
-		{
-			if (IsLightDismissEnabled)
+			void pointerPressed(object sender, Input.PointerRoutedEventArgs args)
```

Comment:
If you put it here as nested method with a capture, I'm not sure that the `-=` will work properly (as the target object will now be the object that captured the `pressed`.

Reply:
Good question.

Reply:
Changed.